ops/services/alerts/app_service_metrics/uptime.tf

```diff
@@ -0,0 +1,74 @@
+locals {
+  geo_locations = [
+    "us-va-ash-azr",   // East US
+    "us-ca-sjc-azr",   // West US
+    "us-il-ch1-azr",   // North Central US
+    "us-tx-sn1-azr",   // South Central US
+    "apac-jp-kaw-edge" // Japan (Guam)
+  ]
+
+  url_prefix = var.env == "prod" ? "www" : var.env
+
+  url_map = {
+    "${var.env}-simplereport-gov"     = "https://${local.url_prefix}.simplereport.gov/",
+    "${var.env}-simplereport-gov-api" = "https://${local.url_prefix}.simplereport.gov/api/actuator/health",
+    "${var.env}-simplereport-gov-app" = "https://${local.url_prefix}.simplereport.gov/app/health/ping"
+  }
+}
+
+resource "azurerm_application_insights_web_test" "uptime" {
+  for_each = local.url_map
+
+  name                    = each.key
+  description             = "Verify the URL is publicly available"
+  location                = var.rg_location
+  resource_group_name     = var.rg_name
+  application_insights_id = var.app_insights_id
+  retry_enabled           = true
+  frequency               = 300
+  timeout                 = 60
+  geo_locations           = local.geo_locations
+  kind                    = "ping"
```

Comment:
What other kinds of web test exist?

Reply:
The only other option is `multi-step` (https://docs.microsoft.com/en-us/azure/azure-monitor/app/availability-multistep) which appears to be on the way out.